### DISCUSSION TITLE
Accessing and importing component max min temperatures using pyAEDT for icepak

### DISCUSSION THREAD

**Discussion by Somayaji1995** [ansys/pyaedt#5049] [Q&A] [answered]
Hi, I have solved model which I am accessing through  pyAEDT script for postprocessing. I want to get the minimum, maximum temperatures from the results in an excel file. Is there any option to get this?

**Comment by Samuelopez-ansys** [comment]
@Somayaji1995  Please play with the create_field_summary method :

fs = icepak_app.post.create_field_summary()
fs.add_calculation("Object", "Volume", "box", "Temperature")
df = fs.get_field_summary_data()

@lorenzovecchietti ANy more hint is welcomed :)

**Comment by Somayaji1995** [comment]
Can we get any examples for this? As it is not working in my case

**Comment by lorenzovecchietti** [comment, marked as answer]
@Samuelopez-ansys 's example should work perfectly.
You can add the `pandas_output` argument:
`df = fs.get_field_summary_data(pandas_output=True)`
and then use pandas method to export to xlsx (https://pandas.pydata.org/docs/reference/api/pandas.DataFrame.to_excel.html)

**Reply by Samuelopez-ansys** [reply]
@Somayaji1995
